Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -220,18 +237,14 @@ class APIClient {
         }
     }
 
-    private func shouldRetry(_ error: Error, operation: AsyncOperation) -> Bool {
+    private func isConnectionError(_ error: Error) -> Bool {
         // We only retry transient errors like connectivity stuff or HTTP 5xx errors
         guard ClientError.isEphemeral(error: error) else {
             return false
         }
 
         let offlineErrorCodes: Set<Int> = [NSURLErrorDataNotAllowed, NSURLErrorNotConnectedToInternet]
```

Comment:
Are these enough for offline error codes? I would suggest going through the URLError.Codes to possibly include others too (e.g. `networkConnectionLost`, `internationalRoamingOff` etc). 
Also, I think it's better to use the more Swifty syntax here (`URLError.Code.notConnectedToInternet`).

Reply:
Actually, this is "improved" in a following PR where only `ClientError.isEphemeral` is used. ATM we are checking codes differently in 3 parts of the LLC...